trellis-api/src/test/java/org/trellisldp/api/NoopNamespaceServiceTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.trellisldp.api;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+public class NoopNamespaceServiceTest {
+
+    @Test
+    public void noAction() {
+        NamespaceService testService = new NoopNamespaceService();
```

Comment:
You should add `final` here.

Reply:
Ah, yes, but also-- the reason you are catching these things and I am not is because my build keeps passing locally. So something is off about my Gradle setup, because it shouldn't be passing this, should it?

It should be running both `checkstyleMain` and `checkstyleTest` (which is what would have gotten this) for the `check` task, right?

Reply:
Correct. Are you running the gradle build from within your IDE or from the command line? And when building, are you running the command at the project root or in a subdirectory?

Reply:
Always CLI, always from project root. Sometimes I will only build over a subdirectory/module (i.e. `gradlew trellis-api:checkstyleMain`).